.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
My point was to not have .npmignore as it's too easy to forget to update it... And i like having src/